lacework/resource_lacework_integration_gcp_at.go

```diff
@@ -162,48 +163,41 @@ func resourceLaceworkIntegrationGcpAtCreate(d *schema.ResourceData, meta interfa
 
 	return resource.RetryContext(context.Background(), d.Timeout(schema.TimeoutCreate), func() *resource.RetryError {
 		retries--
-		log.Printf("[INFO] Creating %s integration\n", api.GcpAuditLogIntegration.String())
-		response, err := lacework.Integrations.CreateGcp(data)
+		log.Printf("[INFO] Creating %s integration\n", api.GcpAtSesCloudAccount.String())
+		response, err := lacework.V2.CloudAccounts.Create(data)
 		if err != nil {
 			if retries <= 0 {
 				return resource.NonRetryableError(
-					fmt.Errorf("Error creating %s integration: %s",
-						api.GcpAuditLogIntegration.String(), err,
+					fmt.Errorf("error creating %s integration: %s",
+						api.GcpAtSesCloudAccount.String(), err,
 					))
 			}
 			log.Printf(
 				"[INFO] Unable to create %s integration. (retrying %d more time(s))\n%s\n",
-				api.GcpAuditLogIntegration.String(), retries, err,
+				api.GcpAtSesCloudAccount.String(), retries, err,
 			)
 			return resource.RetryableError(fmt.Errorf(
-				"Unable to create %s integration (retrying %d more time(s))",
-				api.GcpAuditLogIntegration.String(), retries,
+				"unable to create %s integration (retrying %d more time(s))",
+				api.GcpAtSesCloudAccount.String(), retries,
 			))
 		}
 
-		log.Printf("[INFO] Verifying server response")
-		err = validateGcpIntegrationResponse(&response)
 		if err != nil {
 			return resource.NonRetryableError(err)
```

Comment:
Wondering if this could be moved inside the above if statement?

Reply:
it can be removed entirely, there err it was checking was from the func that was removed

Reply:
Fixed